includes/object-cache.php

```diff
@@ -498,7 +505,7 @@ protected function build_parameters() {
         if ( isset( $parameters['password'] ) ) {
             $password = $parameters['password'];
 
-            if ( is_null( $password ) || $password === '' ) {
+            if ( is_null( $password ) || '' === $password ) {
```

Comment:
Can we turn this Yoda stuff off? I don't like it, even if it is WP's recommendation.